web/impact/impact/settings.py

```diff
@@ -320,6 +320,8 @@ class Test(Base):
     DEBUG = False
     LANGUAGE_CODE = 'en'
 
+    logging.disable(logging.CRITICAL)
```

Comment:
Per ticket, we'd like to see WARNINGs. This looks like we'd see no logging at all.

Reply:
the log messages that show up are [warning messages](https://github.com/masschallenge/django-accelerator/blob/a4123586f3fa90278b222efffc6a944af97540e9/accelerator/managers/member_profile_manager.py#L17) @jonkiparsky , allowing warning shows those as well

Reply:
Okay, then perhaps the issue is that that's not the right log level for that situation. After all, if it's something that happens twice with every test we run, it's probably not a thing we need to be WARNed about, is it? Probably INFO would be sufficient, no?

Reply:
that works for me, this will need another PR. Raising in a few

Reply:
PR raised [here](https://github.com/masschallenge/django-accelerator/pull/183), @jonkiparsky